controllers/storagecluster/storageclasses.go

```diff
@@ -43,7 +43,7 @@ func (obj *ocsStorageClass) ensureCreated(r *StorageClusterReconciler, instance
 		return reconcile.Result{}, err
 	}
 
-	err = r.createStorageClasses(scs)
+	err = r.createStorageClasses(scs, "", instance.Namespace)
```

Comment:
Please add a comment about why we are passing an empty string. Makes it easier to understand.

Reply:
removed